components/layout/display_list/mod.rs

```diff
@@ -1334,6 +1347,14 @@ impl<'a> BuilderForBoxFragment<'a> {
                                 wr::ColorF::WHITE,
                             )
                         }
+
+                        builder.lcp_candidate_collector.update_image_candidate(
+                            self.fragment.base.clone().into(),
```

Comment:
The conversion semantics doesn't quite match here.

Instead of using `into()`, can we simply add a `fn get_lcp_candidate_id()` to `BaseFragment` so that we can avoid cloning it?